documentation/modules/creating-migration-plan.adoc

```diff
@@ -60,6 +60,11 @@ To create a new storage mapping:
 . Select a migration type and click *Next*.
 * Cold migration: The source VMs are stopped while the data is copied.
 * Warm migration: The source VMs run while the data is copied incrementally. Later, you will run the cutover, which stops the VMs and copies the remaining VM data and metadata.
++
+[NOTE]
+====
+{project-short} supports only cold migrations from {osp}.
```

Comment:
we should either add also 'Remote OpenShift Virtualization clusters' here or to say that warm migration is supported only from vSphere and RHV (I think the latter is better)

Reply:
Done.

Reply:
@ahadas That's what I was thinking we should do.